OpenRA.Mods.RA/Traits/GpsWatcher.cs

```diff
@@ -77,12 +83,18 @@ void Refresh()
 		void RefreshInner()
 		{
 			var wasActive = Active;
+			var allyWatchers = owner.World.ActorsWithTrait<GpsWatcher>().Where(kv => kv.Actor.Owner.IsAlliedWith(owner));
```

Comment:
`GpsWatcher` is used on the player actors, so we should be able to cache this, shouldn't we?